app/assets/javascripts/preferences/panes/two-factor-auth/AuthAppInfoPopup.tsx

```diff
@@ -0,0 +1,44 @@
+import { IconButton } from '@/components/IconButton';
+import { FunctionComponent } from 'preact';
+import { useState, useRef, useEffect } from 'react';
+
+/**
+ * AuthAppInfoPopup is an info icon that shows a tooltip when clicked
+ * Tooltip is dismissible by clicking outside
+ *
+ * Note: it can be generalized but more use cases are required
+ * @returns
+ */
+export const AuthAppInfoTooltip: FunctionComponent = () => {
+  const [shown, setShow] = useState(false);
```

Comment:
```suggestion
  const [shown, setShown] = useState(false);
```

Not required of course, I just find it easier to look for stuff if the update function is named after the state variable

Reply:
Changed. I also renamed `shown` to `isShown` for better clarity. 311f50d00cef43129e5fa08819e9987a3625ee87